File: loop.js
const t1 = new Date();
let n = 0;
for (var i = 0; i <1000000000; ++i) {
  n += i;
}
const t2 = new Date();

console.log("t1 : ",t2.valueOf())
console.log("t1 : ",t1.valueOf())
console.log("time : ",t2.valueOf()-t1.valueOf())

File: loop.go
package main

import (
	"fmt"
	"time"
)

func main() {
	t1 := time.Now()

	n := 0

	for i := 0; i < 1000000000; i++ {
		n += i
	}

	t2 := time.Now()

	fmt.Println("t1 : ", t1.UnixMilli())

	fmt.Println("t2 : ", t2.UnixMilli())

	fmt.Println("time : ", t2.UnixMilli()-t1.UnixMilli())
}
